src/main/java/org/apache/maven/shared/utils/logging/MessageUtils.java

```diff
@@ -104,7 +104,7 @@ private static void doSystemUninstall()
 
     /**
      * Enables message color (if JAnsi is available).
```

Comment:
Let's call it "Jansi" as it's done in [their source](https://github.com/fusesource/jansi)

```suggestion
     * Enables message color (if Jansi is available).
```